### PR TITLE
Docs: data source insights

### PR DESCRIPTION
**Pull request by xlson**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/enterprise/usage-insights.md [hunk 1]
```diff
@@ -55,3 +55,17 @@ It shows two kinds of information:
...
 
 {{< docs-imagebox img="/img/docs/enterprise/improved_search.png" max-width="650px" class="docs-image--no-shadow" >}}
+
+## Data source insights
+
+Data source insights gives you information about how much a data source has been used and how many errors it has returned. It shows the last thirty days of data.
+
+To find data source insights, go to:
+1. Data source list view
+2. Click on a data source
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/usage-insights.md:65]: Fix numbering. They should all begin with 1.

File: docs/sources/enterprise/usage-insights.md [hunk 2]
```diff
@@ -55,3 +55,17 @@ It shows two kinds of information:
 In the search view, you can sort dashboards using these insights data. It helps you find unused or broken dashboards or discover most viewed ones.
 
 {{< docs-imagebox img="/img/docs/enterprise/improved_search.png" max-width="650px" class="docs-image--no-shadow" >}}
+
+## Data source insights
+
+Data source insights gives you information about how much a data source has been used and how many errors it has returned. It shows the last thirty days of data.
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/usage-insights.md:61]: Data source insights give you information about how much of the data source has been used and how many errors it has returned. It shows the last thirty days of data.

**Review by achatterjee-grafana — COMMENTED** [review]
Requesting a couple of changes.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

File: docs/sources/enterprise/usage-insights.md [hunk 3]
```diff
@@ -55,3 +55,16 @@ It shows two kinds of information:
...
+Data source insights give you information about how much of the data source has been used and how many errors it has returned. It shows the last thirty days of data.
+
+To find data source insights, go to:
+1. Data source list view
+1. Click on a data source
+1. Click the insights tab
+
+{{< docs-imagebox img="/img/docs/enterprise/datasource_insights.png" max-width="650px" class="docs-image--no-shadow" >}}
+
+> Only available in Grafana Enterprise v7.3+.
```
Comment (AgnesToulet) [inline docs/sources/enterprise/usage-insights.md:70]: I would rather put it at the top, just under the `Data source insights` header.

File: docs/sources/enterprise/usage-insights.md [hunk 4]
```diff
@@ -55,3 +55,16 @@ It shows two kinds of information:
 In the search view, you can sort dashboards using these insights data. It helps you find unused or broken dashboards or discover most viewed ones.
 
 {{< docs-imagebox img="/img/docs/enterprise/improved_search.png" max-width="650px" class="docs-image--no-shadow" >}}
+
+## Data source insights
+
+Data source insights give you information about how much of the data source has been used and how many errors it has returned. It shows the last thirty days of data.
```
Comment (AgnesToulet) [inline docs/sources/enterprise/usage-insights.md:61]:
```suggestion
Data source insights give you access to the following information for the last thirty days:
- Queries per day
- Errors per day
- Average load duration per day (ms)

```

**Review by AgnesToulet — COMMENTED** [review]
I left some suggestions. I found the first sentence a little weird so I tried another style, feel free to accept or discard that change.

File: docs/sources/enterprise/usage-insights.md (hunk 4 above)
Comment (achatterjee-grafana) [reply docs/sources/enterprise/usage-insights.md:61]: I am OK either way. However, I prefer the former as it introduces what the insights feature does.

Comment (xlson) [reply docs/sources/enterprise/usage-insights.md]: I changed it around a little, trying to get the good parts of both ways of writing the docs.

**Pull request merged**